Week04/lesson08/homework/temp_converter/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Temperature Converter - Part 1</title>
+  <link rel="stylesheet" href="css/style.css" />
+</head>
+
+<body>
+  <h1>Temperature Converter</h1>
+  <div id="inputNumber">
+    <input type="number" name="someid" />
```

Comment:
You don't really need the `<div>` around the `<input>`. You know that ultimately, the user types in the `<input>` and that you're going to need to pull the value off of that input. So instead, forget the `<div>` and just give the input an id, maybe like this:

`<input id="c" placeholder="Celsius">` 

(The `placeholder` is that default light grey text inside the input that goes away the minute you click on the input to type in it, not necessary for the JS logic AT ALL, but looks nice on the page)

That said, you're not even going to need user input at all to do the first "version" of this assignment. Details below, in the `index.js` file